src/plugins/mf_priority.cpp

```diff
@@ -1079,6 +1163,12 @@ static int inactive_cb (flux_plugin_t *p,
         b->held_jobs.erase (b->held_jobs.begin ());
     }
 
+    auto it = released_jobs.find (jobid);
+    if (it != released_jobs.end ())
+        // job had an "age" associated with it; remove it from the map that
+        // stores its ID
+        released_jobs.erase (it);
+
```

Comment:
It looks like [std::map erarse()](https://en.cppreference.com/w/cpp/container/map/erase) accepts a key, so could this be replaced with `released_jobs.erase (jobid)`?